### PR TITLE
Remove not nullable params

### PR DESCRIPTION
**Pull request by carusogabriel**
/cc @ondrejmirtes Should PHPStan alert us about this?

### CODE REVIEW

File: packages/RectorBuilder/src/BuilderRector.php [hunk 1]
```diff
@@ -17,17 +17,17 @@
...
-     * @var string|null
+     * @var string
      */
     private $methodName;
 
     /**
-     * @var string|null
+     * @var string
      */
     private $newMethodName;
```
Comment (TomasVotruba) [inline packages/RectorBuilder/src/BuilderRector.php]: What if the Builder matches only a class?

File: src/Rector/Dynamic/Configuration/ArgumentReplacerRecipe.php [hunk 2]
```diff
@@ -110,7 +110,7 @@ public function getType(): string
     }
 
     /**
-     * @return mixed|null
+     * @return mixed
      */
     public function getDefaultValue()
```
Comment (TomasVotruba) [inline src/Rector/Dynamic/Configuration/ArgumentReplacerRecipe.php]: What about `someFunction($argument);`?

File: src/Rector/Dynamic/NamespaceReplacerRector.php [hunk 3]
```diff
@@ -96,7 +96,7 @@ private function resolveNameFromNode(Node $node): string
         }
 
         if ($node instanceof Name) {
-            /** @var FullyQualified|null $resolveName */
+            /** @var FullyQualified $resolveName */
             $resolveName = $node->getAttribute(Attribute::RESOLVED_NAME);
```
Comment (TomasVotruba) [inline src/Rector/Dynamic/NamespaceReplacerRector.php]: What if the value was not resolved for some special case?
It happened before and broke Rector

File: src/Rector/Contrib/Sensio/FrameworkExtraBundle/TemplateAnnotationRector.php [hunk 4]
```diff
@@ -88,7 +88,7 @@ public function isCandidate(Node $node): bool
      */
     public function refactor(Node $classMethodNode): ?Node
     {
-        /** @var Return_|null $returnNode */
+        /** @var Return_ $returnNode */
         $returnNode = $this->betterNodeFinder->findLastInstanceOf((array) $classMethodNode->stmts, Return_::class);
```
Comment (TomasVotruba) [inline src/Rector/Contrib/Sensio/FrameworkExtraBundle/TemplateAnnotationRector.php]: What if no `return` is found? Not every method returns something

File: src/Rector/Contrib/PHPUnit/SpecificMethod/AssertTrueFalseToSpecificMethodRector.php [hunk 5]
```diff
@@ -61,7 +61,7 @@
     private $nodeFactory;
 
     /**
-     * @var string|null
+     * @var string
      */
     private $activeFuncCallName;
```
Comment (TomasVotruba) [inline src/Rector/Contrib/PHPUnit/SpecificMethod/AssertTrueFalseToSpecificMethodRector.php]: This value is null on construct, isn't it?

**Comment by carusogabriel** [comment]
I'll add tests and fix some of them then :sweat_smile:

**Comment by carusogabriel** [comment]
Closing as invalid

**Pull request closed without merging**